OVS_Bonding/README.md

```diff
@@ -0,0 +1,57 @@
+# <u>Setting up network bonding on specific interfaces during the ignition process</u>
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+- All raw files and scripts are located in the RewFiles folder.
+- This guide describes the customisations needed for installation when using a PXE server.
+<br>
+
+## Prerequisites
+- PXE server with ability to see kernel arguments.
+- Web accessible location(HTTP only) to upload the <i> capturemacs.ign </i> file .
+
+## Usage  
+<br>
```

Comment:
can we avoid HTML in this markdown? Let github decide about the looks, we care - first and foremost - about the content.
```suggestion
```

Reply:
Doesn't really matter , Fixed